bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/IDFBuildConfiguration.java

```diff
@@ -117,6 +118,7 @@
 public class IDFBuildConfiguration extends CBuildConfiguration
 {
 
+	private static final String LAUNCH_TARGET_NAME_ATTR = "com.espressif.idf.launch.serial.core.idfTarget"; //$NON-NLS-1$
```

Comment:
This is declared in IDFLaunchConstants.ATTR_IDF_TARGET, can we use the same?

Reply:
Hi @kolipakakondal, yes we can use one from the IDFLaunchConstants. Thank you for noticing it. I forgot we have it there

Reply:
fixed